thirdeye-ui/src/app/components/home-page/alert-accuracy/alert-accuracy.component.tsx

```diff
@@ -70,14 +72,19 @@ export const AlertAccuracy: FunctionComponent<AlertAccuracyProps> = ({
             <Grid item>
                 <LoadingErrorStateSwitch
                     errorState={
-                        <NoDataIndicator>
+                        <NoDataIndicator
+                            className={classNames(classes?.noDataIndicator)}
```

Comment:
If we make classes optional, we should handle the case when its undefined otherwise it should not be optional

Reply:
Wrapping the optional value in `classNames` should take care of that

<img width="827" alt="Screenshot 2022-11-15 at 13 58 49" src="https://user-images.githubusercontent.com/20799363/201868543-1e5819c2-2abc-4d5b-968e-aeb6d4a2d23e.png">

Reply:
I’m more concerned about ‘classes’ not being passed which I know in the current usage will not be the case but it needs to be accounted for. 

So for your particular example when ‘x’ is undefined

Reply:
Added a commit that sets a default value for `classes = {}` in all the instances. 

![Screenshot 2022-11-15 at 15 35 13](https://user-images.githubusercontent.com/20799363/201891228-ecb1668f-a515-428b-9c23-c6ff64a591f5.png)

Added it just in case, but I still don't believe it is required since we are using optional chaining, so `undefined?.undefined` will still be `undefined` without throwing an error.


![Screenshot 2022-11-15 at 15 34 36](https://user-images.githubusercontent.com/20799363/201891104-4ba65c15-468a-4e52-bee0-61b2eea6619e.png)